juego.py

```diff
@@ -386,11 +399,138 @@ def run(self):
 
         self.fondo = cargar_imagen('data/1.jpg')
         self.screen.blit(self.fondo, (0, 0))
-
+        play_again = 0
         pygame.display.flip()
-        while self.running:
+        while self.running or play_again == 1:
+            play_again = 0
             level = self.main()
-            self.play(level)
+            score, puntuacionalta = self.play(level)
+            play_again = self.game_over(score, puntuacionalta)
+
+    def game_over(self, score, high_score):
```

Comment:
There seems to be a repetition of what's been done earlier by `play` like checking for events and handling those events and `MOUSEBUTTONDOWN` seems to be handled by both and `game_over` contains lots of code that I feel it doesn't really need. Why not just render the necessary text when the game is over?

Reply:
I removed unnecessary code. But I'd need to use `MOUSEBUTTONDOWN` as there are `Play-again` and `Exit` buttons in `game_over`. Also, I deleted this branch by mistake. So will it be okay to make another PR with changes and close this one? Or is there a way to push commits from a new branch to this PR?